docs/clusterdefinition.md

```diff
@@ -131,6 +132,22 @@ More usefully, let's add some custom configuration to both of the above addons:
                   "memoryLimits": "512Mi"
                 }
               ]
+        },
+        {
+            "name": "cluster-autoscaler",

```

Comment:
nit: line 106 above says `More usefully, let's add some custom configuration to both of the above addons:` , could you please change that to ``More usefully, let's add some custom configuration to the above addons:` since there are now 3 examples?

Reply:
Done.